CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```

Comment:
should this be changed?

Reply:
I plan to create a separate release PR that updates that unreleased section. 

This diff is just to update the changelog in preparation for the release since the release instructions have specific steps for creating a release branch from the state of dev.